tests/Koans16AboutTime.elm

```diff
@@ -1,35 +1,55 @@
 module Koans16AboutTime exposing (testSuite)
 
 import Expect
-import Utils.Test exposing (describe, test)
+import Time exposing (..)
```

Comment:
Can we use the qualified import for `Time` here?

Reply:
Sure - thanks.